CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 We [keep a changelog.](http://keepachangelog.com/)
 
+## 1.13.0
+
+As part of extending our CLI, we have been migrating to a plugin-based system. 
+This release removes the `anaconda` entrypoint from `anaconda-client`, which is now located instead in `anaconda-cli-base`.
+In order to maintain backwards compatibility for all subcommands, we include logic to mount all existing subcommands from `anaconda-client` as top-level subcommands. 
```

Comment:
how about "... as hidden top-level subcommands. anaconda-client subcommands continue to work until overridden by new plugins, but will not be displayed in the help output"

Reply:
Thank you, I've modified the text accordingly.